gradle.properties

```diff
@@ -50,3 +50,5 @@ datadogClientToken=do_not_change_here
 datadogEnvironment=qa
 datadogServiceName=simple-android
 maestroTests=false
+org.gradle.unsafe.configuration-cache=true
+org.gradle.unsafe.configuration-cache-problems=warn
```

Comment:
Let's fallback to default severity which is error. If there is any task that is throwing errors, let's resolve them if we can.

Reply:
Fixed